Backend/app/dashboard/CommonAPIs/src/Dashboard/ProviderPlatform/Merchant.hs

```diff
@@ -256,16 +257,17 @@ type DriverIntelligentPoolConfigUpdateAPI =
     :> ReqBody '[JSON] DriverIntelligentPoolConfigUpdateReq
     :> Post '[JSON] APISuccess
 
+-- FIXME check for new fields added (for all configs)
 data DriverIntelligentPoolConfigUpdateReq = DriverIntelligentPoolConfigUpdateReq
-  { availabilityTimeWeightage :: Maybe Int,
-    availabilityTimeWindowOption :: Maybe SWC.SlidingWindowOptions,
-    acceptanceRatioWeightage :: Maybe Int,
+  { availabilityTimeWeightage :: Maybe (MandatoryValue Int),
+    availabilityTimeWindowOption :: Maybe SWC.SlidingWindowOptions, -- value wrapper make no sense for lists and objects
```

Comment:
do we need also use value wrapper for lists and objects?

Reply:
What is we have to update this value to null?